### PR TITLE
#157505126 Set up linting, Wiki and Readme

### PR DESCRIPTION
**Pull request by Victoradukwu**
#### What does this PR do?
- Set up **pylint** , readme file and project wiki
#### Description of Task to be completed?
- Install pylint to lint code in accordance with PEP8
- Add a readme file
- Add a project Wiki to the Github repo
#### How should this be manually tested?
- Clone the application
- Run `pip install -r requirements.txt` to install all dependencies
- Add a python file and observe the linter pointing out where there is a deviation from the team convention. 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 - ch/157505126/code conventions + linter + readme
#### Screenshots (if appropriate)
-N/A
#### Questions:
N/A

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+    ```
+    source venv/bin/activate
+    ```
+- Install dependencies
+    ```
+    pip install -r requirements.txt
+    ```
+- Create Application environment variables and save them in .env file
+    ```
+    export APP_SETTINGS="Development" # set app Enviroment.
```
Comment (ibutiti) [inline Readme.md]: Hey man. Please check the PR I made for the env variables needed.

File: Readme.md [hunk 2]
```diff
@@ -1,3 +1,87 @@
...
+    export SECRET_KEY="some-very-long-string-of-random-characters"
+    export DEV_DATABASE_URL="" # Db for Development.
+    export TEST_DATABASE_URL="" # Db for Testing
+    export DATABASE_URL="" # Db for Production
+    ```
+- Running migrations
+
+    - Migrate database to new structure. Run the command below:
```
Comment (ibutiti) [inline Readme.md]: Also, using `flask cli` and `flask migrate`.
Initializing db is:
`flask init db`

Creating migrations:
`flask db migrate`

executing migrations:
`flask db migrate`

Comment (kodero) [reply Readme.md]: I like this @ibutiti. @Victoradukwu, lets incorporate that.

File: Readme.md (hunk 1 above)
Comment (Victoradukwu) [reply Readme.md]: Please provide access to a dummy .env file. @ibutiti

**Comment by Victoradukwu** [comment]
@Dheavyman Thanks. I have incorporated that based on @ibutiti review

**Comment by Victoradukwu** [comment]
Changes implemented, @ibutiti @Dheavyman @kodero

**Comment by kodero** [comment]
@Victoradukwu, can you work on the merge conflicts?

**Pull request merged**